README.md

```diff
@@ -176,6 +176,70 @@ Releases and publishes the package. Runs tests, lints and builds the project bef
 
 This command uses [`standard-version`](https://github.com/conventional-changelog/standard-version) underneath. The version is automatically inferred from the [conventional commits](https://conventionalcommits.org/).
 
+## Using a linked version of [@discussify-styleguide](https://github.com/ipfs-shipyard/discussify-styleguide)
+
+In some cases, you may want to make changes to Discussify's styleguide at the same time as you work on your project which uses the styleguide. In order to use a local version of [@discussify-styleguide](https://github.com/ipfs-shipyard/discussify-styleguide) and have any styleguide modifications be reflected live on your project, some pages have to be made in our main project.
+
+### Install `postcss-import-webpack-resolver`
+
+Run the following command in your main project
+
+```sh
+$ npm i postcss-import-webpack-resolver
+```
+
+### Update `postcss-preset-moxy` to the latest version
+
+In `package.json` replace the version of the `postcss-preset-moxy` dependency to be equal to or above `^3.0.0`.
+
+### Make required changes in `webpack.js`.
+
+Add two new dependencies. `fs` will be used to check if the styleguide's version is linked or normally installed and `postcss-import-webpack-resolver` will allow the creation of aliases in postcss.
```

Comment:
I would simplify this a lot, there’s no need to explain in so much detail. Also we are assuming we have a `buildConfig` function within all apps using the styleguide, which might not be the case. Adjust all of this like you don’t know how the Webpack config is built.